lib/helpers/config.js

```diff
@@ -29,22 +29,21 @@ var Config = module.exports = function(folder) {
 		"dependencies": null,
 		"watch": null
 	}
-	var parsers = [Yaml, Json]
-	this.parser = this._getParser(parsers)
+	this.parsers = [Yaml, Json]
 }
 
-/**
- * Decide which parser will be used
- * @return {Object} parser
- */
-Config.prototype._getParser = function(parsers) {
+Config.prototype._getParsedConfig = function (local) {
+	var baseFilename = local ? CONFIG_FILENAME_LOCAL : CONFIG_FILENAME
 	var returning = null
-	parsers.forEach((function(parser) {
-		if(fs.existsSync(path.resolve(this.dir, CONFIG_FILENAME + '.' + parser.getExtension()))) returning = parser
-	}).bind(this))
+	this.parsers.forEach(function(parser) {
+		parser.getExtensions().forEach(function (extension) {
+			var resolvedPath = path.resolve(this.dir, baseFilename + '.' + extension)
+			if(!returning && fs.existsSync(resolvedPath)) {
+				returning = parser.get(resolvedPath)
+			}
+		}.bind(this))
+	}.bind(this))
 	if(returning) return returning
-	log.error('Cannot find mango config file')
```

Comment:
Ne, jinak. Vyhodí to error z řádku 68 "Cannot load the config file mango.json or mango.yaml". @enzy  @Onset